frontend/src/components/auth/Signup.tsx

```diff
@@ -47,8 +47,6 @@ const Signup = (): React.ReactElement => {
   if (authenticatedUser) {
     return <Redirect to={HOME_PAGE} />;
   }
-  // Lock scroll
-  document.body.style.overflow = "hidden"
   return (
     <Flex h="100vh">
         <Box w="47%">
```

Comment:
The vertical and horizontal overflow is happening on this page as well. The same solution of changing the widths so the total is less than 100% works.